content/en/Platform Deep Dive/Collaboration/Organization/_index.md

```diff
@@ -19,3 +19,25 @@ You can view the organization you belong to in the Cobalt app. If you joined mul
 {{< alert title="Note" color="primary" >}}
 Your user role determines your set of privileges within an organization or [pentest](/platform-deep-dive/pentests/). Learn more about [user roles and permissions](/platform-deep-dive/collaboration/organization/user-roles/).
 {{< /alert >}}
+
+## Download Your Organization's Pentest Data
+
+To export pentest data for your organization, you can:
+
+- Use the Cobalt API. Read our [API documentation](https://docs.cobalt.io/v2/) for details. You can send API requests to retrieve pentest data, such as:
```

Comment:
@eugenerevzin is there anything with the Workato integration that you plan to broadcast? I saw the recipe for findings into google sheets? This could be a potential future add on here on how customers can download their data. https://docs.google.com/presentation/d/1N4QPUy9mmZTf-NTJfvaz_gVuIrwKqoz40zsjq0Ro_bA/edit#slide=id.g1fc8a7f90ee_1_144